src/qt/pivx/settings/forms/settingsfaqwidget.ui

```diff
@@ -1416,7 +1416,7 @@
                       <item>
                        <widget class="QLabel" name="labelSubtitle9">
                         <property name="text">
-                         <string>What is a Master Node?</string>
```

Comment:
this is changed in https://github.com/PIVX-Project/PIVX/pull/1184 along with other occurrences (there's 4 only in this file)

Reply:
rebased after merge of #1184